src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -239,6 +239,16 @@ class PackageDetailTab extends mixin(StoreMixin) {
     );
   }
 
+  getTermsConditionUrl() {
+    const cosmosPackage = CosmosPackagesStore.getPackageDetails();
+
+    if (cosmosPackage.selected) {
```

Comment:
Replace with `cosmosPackage.isCertified()`? Looks like that is used elsewhere in this file, and is the more readable option.